tests/tinderbox_scraper/test_invalid_date_tinderbox.py

```diff
@@ -6,36 +6,24 @@
 
 from mozdownload import TinderboxScraper
 
-import mozhttpd_base_test as mhttpd
-
-
-# testing with an invalid date parameter should download the latest build for
-# localized builds
-tests_with_invalid_date = [
-    # -a firefox -p win32 --branch=mozilla-central --date=20140317030202
-    {'args': {'application': 'firefox',
-              'branch': 'mozilla-central',
-              'date': '20140317030202',
-              'locale': 'pt-PT',
-              'platform': 'win32'}},
-    # -p win32 --branch=mozilla-central  --date=invalid
-    {'args': {'branch': 'mozilla-central',
-              'date': 'invalid',
-              'locale': 'pt-PT',
-              'platform': 'win32'}},
-    # -p win64 --branch=mozilla-central --date=2013/07/02
-    {'args': {'branch': 'mozilla-central',
-              'date': '2013/07/02',
-              'platform': 'win64'}},
-    # -p win32 --branch=mozilla-central --date=2013-March-15
-    {'args': {'branch': 'mozilla-central',
-              'date': '2013-March-15',
-              'platform': 'win32'}}
-]
-
-tests = tests_with_invalid_date
+import pytest
 
 
+@pytest.mark.parametrize("args", [
+    ({'application': 'firefox', 'branch': 'mozilla-central', 'date': '20140317030202', 'locale': 'pt-PT', 'platform': 'win32'}),
```

Comment:
nit: this line exceeds the 100 char line limit as shown by the linter check. Please wrap the line.